vscode-smart-contracts/package.json

```diff
@@ -42,6 +42,10 @@
       {
         "command": "concordium-smart-contracts.test",
         "title": "Concordium Smart Contracts: Test contract"
+      },
+      {
+        "command": "concordium-smart-contracts.setup-project",
+        "title": "Concordium Smart Contracts: Setup new smart contract project"
```

Comment:
Is using "Setup" as the wording the common thing to do for this sort of command?

Why not use "initialize" like the cargo command?

Reply:
No particular reason. I'll change this, since it makes sense to follow the same naming